docker-entrypoint.sh

```diff
@@ -126,19 +126,22 @@ if [ "$1" = 'rabbitmq-server' ]; then
 			].
 		EOF
 	fi
+fi
 
-	if [ "$ssl" ]; then
-		# Create combined cert
-		cat "$RABBITMQ_SSL_CERT_FILE" "$RABBITMQ_SSL_KEY_FILE" > /tmp/combined.pem
-		chmod 0400 /tmp/combined.pem
+if [ "$ssl" ]; then
+	# Create combined cert
+	# If calling this script with an exec this is problematic, as it needs to be owned 400 for perms but this code can run as root OR rabbitmq.
+	# As a workaround prefix the combined cert with the username so each user only operates on the file they control.
+	combined_file="/tmp/$(id -un)_combined.pem"
```

Comment:
Well, I suppose though I personally don't care for it as omitting the user will break other users if the file becomes owned with a different UID than the user's.  That was the issue that prompted the commits we're discussing, so I'd like to keep them.  In thinking about this I think I've lost the trail somewhat as, per other comments, it's clear that the '-u ${user}' is something that should be supported, as such my last change isn't good and I plan to revert it.  With those things in mind, though, I'd like to switch back to my original idea of creating a file for each user so there are no permissions issues and so the user isn't tied to a single UID.  To resolve @yosifkit's original concern I'm thinking of switching from the username to the numeric user ID which will always be available.  This should work for any value passed in on '-u'.  Before I make these changes, is that an agreeable course of action?  I think it's a solution that will work in the most use cases.